internal/process/process.go

```diff
@@ -4,12 +4,10 @@
 package process
 
 import (
-	"strings"
-	"sync"
-	"time"
-
 	"github.com/newrelic/infra-integrations-sdk/data/metric"
 	"github.com/newrelic/nri-vsphere/internal/load"
+	"strings"
+	"sync"
```

Comment:
this is wired , i think vscode puts std packages first. not an issue but we should check this